packages/testing-library/examples/basic/package.json

```
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "rspeedy build",
     "dev": "rspeedy dev",
-    "test": "vitest"
+    "test": "vitest",
+    "test:rs": "rstest run"
```

nitpick: 

```suggestion
    "rstest": "rstest"
```

or 
```suggestion
    "test:rstest": "rstest"
```


I personally think the `:rs` here doesn't intuitively indicate `rstest`, instead it gives me the impression of testing Rust. Also, the `run` should be unnecessary.